lib/kafo/kafo_configure.rb

```diff
@@ -371,6 +369,11 @@ def validate_all(logging = true)
 
     def run_installation
       self.class.hooking.execute(:pre)
+
+      hiera = HieraConfigurer.new(config.app[:hiera_config], config.modules, config.app[:order])
+      hiera.write_configs
```

Comment:
right. I was confused by `generate_config` but I see now it works under copy of original file